### PR TITLE
begin v3 era contributing documentation

### PR DESCRIPTION
**Pull request by efx**
My beliefs on documentation have been strongly influenced by Daniele Procida's system [here](https://documentation.divio.com/).
In that framework, our docs/CONTRIBUTING.md acts as a reference document.
We can then write individual how to guides for discrete tasks. Writing a concept exercise is one good example.

I removed the stale `Contributing` section in the README too.
I'll plan on chipping away at this PR this week!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,15 +7,7 @@ Exercism exercises in Rust
...
-
-We welcome pull requests of all kinds. No contribution is too small.
-
-We encourage contributions that provide fixes and improvements to existing exercises. Please note that this track's exercises must conform to the Exercism-wide standards described in the [documentation](https://github.com/exercism/docs/tree/main/language-tracks/exercises). If you're unsure about how to make a change, then go ahead and open a GitHub issue, and we'll discuss it.
-
-If you want to contribute but don't know where to start, the [issues](https://github.com/exercism/rust/issues) tab shows what items are currently of interest to the community. In addition, it is often worth checking the [track dashboard](https://tracks.exercism.io/rust/main), which flags any exercises whose version has fallen behind the version of the [canonical data](https://github.com/exercism/problem-specifications). PRs which keep the track up to date are particularly welcome!
+Check out our [contributor documentation](docs/contributing).
```
Comment (coriolinus) [inline README.md:10]:
```suggestion
Check out our [contributor documentation](docs/CONTRIBUTING.md).
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- improving existing exercises
+- creating new exercises
+- internal tooling
+
+## Merging Philosophy
+
+A [pull request](https://docs.github.com/en/github/getting-started-with-github/github-glossary#pull-request) should address one logical change.
+This could be small or big.
+
+For example, [#1174](https://github.com/exercism/rust/pull/1174) fixed a single typo in a single file after minutes of collaboration.
```
Comment (coriolinus) [inline docs/CONTRIBUTING.md:20]:
```suggestion
For example, [#1175](https://github.com/exercism/rust/pull/1175) fixed a single typo in a single file after minutes of collaboration.
```

Fairly sure that was just an off-by-one error.

Comment (efx) [reply docs/CONTRIBUTING.md:20]: Perfect, thank you!

**Comment by efx** [comment]
I'm ready for your thoughts on a WIP version of this to merge. I'm copying JavaScript's approach in saying [our contributing documentation needs updated.](https://github.com/exercism/javascript/blob/main/CONTRIBUTING.md#contributing)

I quite understand if you would prefer to flesh out the document more before merging. My feelings are that I've added what I want from a high level. My next step would be to write a how-to guide for creating a new concept exercise based on what we've learned over the past year. Then, I can link to that from here and our other issues to start new contributors on the right foot.

After that, I'll try to get back to the actual concepts documentation and exercise work!

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Contributing to the Rust Exercism Track
+
+This document is a work-in-progress.
+Current maintainers are primarily preparing for the v3 launch of the Rust track.
```
Comment (coriolinus) [inline docs/CONTRIBUTING.md:4]: This line will get stale fast. One way we might future-proof it is by phrasing the paragraph like:

This track is a work in progress. Please take all the value you can from it, but if you notice any way to improve it, we are eager for pull requests. Fixing a typo in documentation is every bit as valid a contribution as a massive addition of code.

> A work of art is never finished, merely abandoned.
>
> -- Paul Valéry

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+Nonetheless, feel free to peruse what we have written thus far!
+
+*Contributions welcome :)*
+
+Thank you so much for contributing! :tada:
```
Comment (coriolinus) [inline docs/CONTRIBUTING.md:9]: Though this line was inherited from previous documentation, I feel that it fits poorly here. Probably the best options might be to move it back to the first non-header line of the document, or remove it.

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+As one of the many tracks in Exercism, contributions here should observe Exercism standards like the [Code of Conduct](https://exercism.io/code-of-conduct).
+This document introduces the ways you can help and what maintainers expect of contributors.
+
+## Ways to Contribute
+
+As with many Open Source Projects, work abounds.
```
Comment (coriolinus) [inline docs/CONTRIBUTING.md:15]:
```suggestion
As with many Open Source projects, work abounds.
```

I don't think that "Projects" deserves capitalization here; it's not really part of the phrase.

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+Here are a few categories of what contributions:
```
Comment (coriolinus) [inline docs/CONTRIBUTING.md:16]:
```suggestion
Here are a few categories of welcome contribution:
```

Clarify grammar.

Also might be worth adding:

- fixing typos, misspellings, and grammatical errors
- updating documentation

**Review by coriolinus — APPROVED** [review]
A few notes. I leave it to your discretion which to implement before merging.

File: docs/CONTRIBUTING.md (hunk 4 above)
Comment (efx) [reply docs/CONTRIBUTING.md:9]: Great point. I feel removing is best!

File: docs/CONTRIBUTING.md (hunk 3 above)
Comment (efx) [reply docs/CONTRIBUTING.md:4]: I love the Paul Valéry quote. Frames our work quite wonderfully.

**Pull request merged**